src/sdk-reference/js/6/kuzzle-event-emitter/prependOnceListener/snippets/prepend-once-listener.js

```diff
@@ -0,0 +1,11 @@
+const eventEmitter = new KuzzleEventEmitter();
```

Comment:
I think the following example is clearer:

```js
const eventEmitter = new KuzzleEventEmitter();

eventEmitter.addListener('event', () => console.log('listener1'));

eventEmitter.prependOnceListener('event', () => console.log('listener2'));

// Prints:
//   listener2
//   listener1
eventEmitter.emit('event');

// Prints: listener1
// (the listener function printing "listener2" as since been removed)
eventEmitter.emit('event');
```